link/src/main/java/com/stripe/android/link/LinkConfigurationInteractor.kt

```diff
@@ -0,0 +1,120 @@
+package com.stripe.android.link
+
+import android.content.Context
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.injection.ENABLE_LOGGING
+import com.stripe.android.core.injection.IOContext
+import com.stripe.android.core.injection.NonFallbackInjectable
+import com.stripe.android.core.injection.PUBLISHABLE_KEY
+import com.stripe.android.core.injection.STRIPE_ACCOUNT_ID
+import com.stripe.android.core.injection.UIContext
+import com.stripe.android.core.networking.AnalyticsRequestExecutor
+import com.stripe.android.link.injection.DaggerLinkPaymentLauncherComponent
+import com.stripe.android.link.injection.LinkPaymentLauncherComponent
+import com.stripe.android.link.ui.inline.UserInput
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.networking.PaymentAnalyticsRequestFactory
+import com.stripe.android.networking.StripeRepository
+import com.stripe.android.payments.core.injection.PRODUCT_USAGE
+import com.stripe.android.uicore.address.AddressRepository
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.flatMapMerge
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.coroutines.CoroutineContext
+
+@Singleton
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class LinkConfigurationInteractor @Inject internal constructor(
```

Comment:
Why can't this be a part of LinkAccountManager?

Reply:
LinkAccountManager has a fixed config. This config changes. And we observe those changes in the UI. This is kinda the coordinator of that.

So, maybe `LinkConfigurationCoordinator`?